src/wasm.h

```diff
@@ -59,6 +59,7 @@
 #include "pretty_printing.h"
 #include "support/bits.h"
 #include "support/utilities.h"
+#include "support/metrics.h"
```

Comment:
Then does #208 require this addition to `wasm.h`? Could we remove it from here and have it only there, if so? I'd hope we don't need this even there, though.